src/main/java/com/redhat/red/build/finder/report/GAVReport.java

```diff
@@ -16,18 +16,20 @@
 package com.redhat.red.build.finder.report;
 
 import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
 import java.util.stream.Collectors;
 
 import com.redhat.red.build.finder.KojiBuild;
 import com.redhat.red.build.koji.model.xmlrpc.KojiBuildInfo;
 
 public class GAVReport extends Report {
-    private List<String> gavs;
+    private Set<String> gavs;
 
     public GAVReport(List<KojiBuild> builds) {
-        List<KojiBuildInfo> buildInfos = builds.stream().filter(b -> b.getType() != null && b.getType().contains("maven")).map(KojiBuild::getBuildInfo).collect(Collectors.toList());
-        this.gavs = buildInfos.stream().map(b -> b.getMavenGroupId() + ":" + b.getMavenArtifactId() + ":" + b.getMavenVersion()).collect(Collectors.toList());
-        this.gavs.sort(String::compareToIgnoreCase);
+        // FIXME: Will check that !b.getTypes().contains("rpm") instead of getBuildInfo().getMavenArtifactId().endsWith("-repolib"), but requires NOS-1152, NOS-1153
+        List<KojiBuildInfo> buildInfos = builds.stream().filter(b -> b.getTypes() != null && b.getTypes().contains("maven") && !b.getBuildInfo().getMavenArtifactId().endsWith("-repolib")).map(KojiBuild::getBuildInfo).collect(Collectors.toList());
```

Comment:
Do we have a tracker jira to fix this up later?

Reply:
I think the workaround is OK as in practice as the only RPMs that have maven artifacts are named '-repolib' by convention. 

This issue has the Kojiji-related issues linked to it. I am not sure if there's a better way to do it.

Reply:
It's the same instance that tracks the tickets for this.